src/awkward/_connect/rdataframe/from_rdataframe.py

```diff
@@ -246,7 +246,16 @@ def form_dtype(form):
         else:
             contents[key] = value
 
+    out = ak.zip(contents, depth_limit=1)
+
+    if keep_order:
+        sorted = ak.index.Index64(contents["rdfentry_"].data.argsort())
```

Comment:
I don't think we need to specify a sorting algorithm here because the `rdfentry_` values are always unique.

Reply:
Right, that makes sense.